package-res/resources/web/prompting/api/OperationAPI.js

```diff
@@ -54,18 +54,18 @@ define(['common-ui/prompting/PromptPanel', 'common-ui/prompting/parameters/Param
      *                                    or asynchronously.
      * @example
      *  // Asynchronous
-     *  api.operation.render("prompt-panel-render-area", function(api, callback) {
+     *  api.operation.render(function(api, callback) {
      *    retrieveXml(function(xml){
      *      callback(xml);
      *    });
      *
      *  // Synchronous
-     *  api.operation.render("prompt-panel-render-area", function(api, callback) {
+     *  api.operation.render(function(api, callback) {
      *    var xml = retrieveXml();
      *    callback(xml); // return already retrieved xml definition
      *  });
      */
-    this.render = function(id, getParameterXml) {
+    this.render = function(getParameterXml) {
```

Comment:
Need remove unused parameter in js doc: "@param {String}    id HTML object id where the render the panel"